k8s/distributed-chroma/templates/compaction-service.yaml

```diff
@@ -46,6 +46,8 @@ spec:
           image: "{{ .Values.compactionService.image.repository }}:{{ .Values.compactionService.image.tag }}"
           imagePullPolicy: IfNotPresent
           readinessProbe:
+            periodSeconds: 1
+            failureThreshold: 30 # 30 seconds to allow for initial startup
```

Comment:
the overall timeout remains unchanged from the default, since the default is periodSeconds: 10 & failureThreshold: 3